src/views/AdminComponent/AdminTableContainer/InternshipTable.js

```diff
@@ -1,37 +1,75 @@
-import React, { Fragment, useState, useEffect } from 'react'
+import React, { Fragment, useState, useEffect, createRef} from 'react'
 import { useTable, useSortBy, useExpanded, usePagination } from 'react-table'
-
+import axios from 'axios'
+import {CSVLink} from 'react-csv'
 
 import "bootstrap/dist/css/bootstrap.min.css"
 // reactstrap component
 import {
-    Button,
-    Card, CardBody,
+    Button, Container,
+    Card, CardHeader, CardBody,
+    NavItem, NavLink, Nav, Progress,
     Table, Row, Col,
-    CustomInput,
-    Modal, ModalBody,
+    Input, CustomInput,
+    Modal, ModalHeader, ModalBody, ModalFooter,
+    Form, FormGroup, Label, FormText
 } from "reactstrap";
 
-import Apply from '../../../components/Modal/ApplyForm'
-import Add from '../../../components/Modal/AddForm'
+import AddIntern from '../../../components/Modal/AddInternForm'
 
 import { ReactComponent as DownloadIcon } from '../../../assets/img/icons/common/save_alt_white_24dp.svg'
 import { ReactComponent as DeleteIcon } from '../../../assets/img/icons/common/delete_white_24dp.svg'
 
-import { tablestyle, applybtnshadow, headingstyle, expandbgstyle } from '../../../components/Style/css_style'
+import { tablestyle, headingstyle, expandbgstyle } from '../../../components/Style/css_style'
+
+function getCookie(name) {
+    var cookieValue = null;
+    if (document.cookie && document.cookie != '') {
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+            var cookie = cookies[i].trim();
+            if (cookie.substring(0, name.length + 1) == (name + '=')) {
+                cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                break;
+            }
+        }
+    }
+    return cookieValue;
+}
+
+const base_url = 'http://127.0.0.1:8000'
+
+const TableContainer = ({ columns, data, changeData }) => {
+
+    const [fetchedData, setFetchedData] = useState([])
+    const [changed, setChanged] = useState(true)
+    const [csvLink, setCsvLink] = useState(React.createRef())
+    const [expand, setExpand] = useState(true)
+
+    useEffect(async() => 
```

Comment:
you are getting a warning here. If you have time kindly clear all warnings and fix indentation you can use prettier for this.

Reply:
I have cleared all those warnings which I could and which did not affect the model.